inference-engine/src/mkldnn_plugin/ngraph_transformations/convert_to_power_static.cpp

```diff
@@ -89,8 +95,8 @@ std::shared_ptr<ngraph::Node> convert(const std::shared_ptr<BaseOp> &node) {
 NGRAPH_RTTI_DEFINITION(MKLDNNPlugin::ConvertToPowerStatic, "ConvertToPowerStatic", 0);
 
 MKLDNNPlugin::ConvertToPowerStatic::ConvertToPowerStatic() {
-    ngraph::OutputVector twoInputs = {ngraph::pattern::any_input(ngraph::pattern::has_static_shape()),
-                                      ngraph::pattern::any_input(ngraph::pattern::has_static_shape())};
+    ngraph::OutputVector twoInputs = {ngraph::pattern::any_input(ngraph::pattern::has_static_rank()),
```

Comment:
Can we add new tests?

Reply:
I know that @jane-intel locally verified this simple changes and I think we can make it in a separate PR since this CPU specific transformations wasn't covered by tests and we don't have any target where we can add such tests. So I would create ticket for CPU plugin to make them cover this transformations with tests.